plugins/inputs/sql/README.md

```diff
@@ -0,0 +1,192 @@
+# SQL plugin
+
+The plugin executes simple queries or query scripts on multiple servers.
+It permits to select the tags and the fields to export, if is needed fields can be forced to a choosen datatype. 
+Supported/integrated drivers are mssql (SQLServer), mysql (MySQL), postgres (Postgres)
+Activable drivers (read below) are all golang SQL compliant drivers (see https://github.com/golang/go/wiki/SQLDrivers): for instance oci8 for Oracle or sqlite3 (SQLite)
+
+## Getting started :
+First you need to grant read/select privileges on queried tables to the database user you use for the connection
+
+### Non pure go drivers
+For some not pure go drivers you may need external shared libraries and environment variables: look at sql driver implementation site
+Actually the dependencies to all those drivers (oracle,db2,sap) are commented in the sql.go source. You can enable it, just remove the comment and perform a 'go get <driver git url>' and recompile telegraf. As alternative you can use the 'golang 1.8 plugins feature'like described here below
+
+### Oracle driver with golang 1.8 plugins feature
+Follow the docu in https://github.com/mattn/go-oci8 for build the oci8 driver.
+If all is going well now golang oci8 driver is compiled and linked against oracle shared libs. But not linked in telegraf.
+
+For let i use in telegraf, do the following:
+create a file plugin.go with this content:
+
+```
+package main
+
+import "C"
+
+import (
+	"log"
+	// .. here you can add import to other drivers
+	_ "github.com/mattn/go-oci8" // requires external prorietary libs
+	// _ "bitbucket.org/phiggins/db2cli" // requires external prorietary libs
+	// _ "github.com/mattn/go-sqlite3" // not compiles on windows
+)
+func main() {
+	log.Printf("I! Loaded plugin of shared libs")
+}
+``` 
+build it with
+``` 
+mkdir $GOPATH/lib
+go build -buildmode=plugin -o $GOPATH/lib/oci8_go.so plugin.go
+```
+in the input plugin configuration specigy the path of the created shared lib 
+``` 
+[[inputs.sql]]
+	...
+	driver = "oci8"
+	shared_lib = "/home/luca/.gocode/lib/oci8_go.so"
+	...
+``` 
+
+The steps of above can be reused in a similar way for other proprietary and non proprietary drivers
+
+
+## Configuration:
+
+```
+	[[inputs.sql]]
```

Comment:
In all example configs, alight the header to the left and use two space indention of the body, place comments above the option with double `##`, single comment out optional fields (including those with defaults), and only required fields should be uncommented.